### ISSUE TITLE
[Task] v-overflow-btn

### ISSUE THREAD

**Issue by chris-saburo** [vuetifyjs/vuetify#13493]
# Overview

- Related Tasks: n/a
- Related Issues: n/a

A conversion should not start until *all* assigned issues with the milestone v2.2.x are resolved and in the `next` branch. Reference our [Coding Guidelines](https://github.com/vuetifyjs/vuetify/wiki/coding-guidelines) for information regarding the team's practices or reach out to us on [Discord](https://community.vuetifyjs.com).

**Comment by johnleider** [comment]
Unfortunately this is being pushed to a labs release later this year. Sorry for an inconvenience.

**Comment by J-Sek** [comment]
I have never used this component in v2. When I click old docs it feels clunky, unpolished and kinda redundant. Would a Split Button (from MD3) be a natural replacement? I am curious what are the use cases in existing legacy applications.
- quick demo for [VSplitBtn](https://play.vuetifyjs.com/playgrounds/ik437Q)

**Comment by johnleider** [comment]
> I have never used this component in v2. When I click old docs it feels clunky, unpolished and kinda redundant. Would a Split Button (from MD3) be a natural replacement? I am curious what are the use cases in existing legacy applications.
> 
> * quick demo for [VSplitBtn](https://play.vuetifyjs.com/playgrounds/ik437Q)

I'm okay with this, especially if there is something in the spec.

**Comment by J-Sek** [comment]
closing in favor of #22076
